habit-backend/habit/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include('users.urls')),
```

Comment:
Where does this get `users` from?
Do we not have to import users model?

Reply:
This means from users directory include urls.py

Reply:
and this users directory is not a relative link, right?

Reply:
Yes. You can think of it as a file which contain all the endpoints of the project. Here it will import the end point from all the apps of the project.